README.md

```diff
@@ -98,29 +98,29 @@ For more detailed example code that shows more methods, have a look at the [test
 
 Constructs an instance of the graph data structure.
 
-The optional argument _serialized_ is a serialized graph that may have been generated by **[serialize](#serialize)**. If _serialized_ is present, it is deserialized by invoking **[deserialize](#deserialize)**.
+The optional argument _serialized_ is a serialized graph that may have been generated by **[serializeGraph](#serializeGraph)**. If _serialized_ is present, it is deserialized by invoking **[deserializeGraph](#deserializeGraph)**.
```

Comment:
Nope, to deserialize the graph you have to call `deserializeGraph(mySerializedObject)`. There constructor takes no parameter anymore